lib/mayaUsd/undo/UsdUndoStateDelegate.cpp

```diff
@@ -101,7 +118,13 @@ void UsdUndoStateDelegate::invertPushTokenChild(
     TF_DEBUG(USDMAYA_UNDOSTATEDELEGATE)
         .Msg("Inverting push field '%s' of '%s'\n", fieldName.GetText(), parentPath.GetText());
 
+#if 1
```

Comment:
Debugging code?

Reply:
@ppt-adsk good point. they can go.  addressed in 0dc4dd0409338b54ef30f664471ec7e3981b5ffa